code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -35,7 +35,8 @@
 
 	return 0
 
-///mob/living/carbon/alien/humanoid/bullet_act(obj/item/projectile/Proj) taken care of in living
+/mob/living/carbon/alien/humanoid/CanContractDisease(datum/disease/D)
```

Comment:
Why have this on the humanoid level and not just the alien level? Surely it can't hurt to cover more subtypes here.